src/algorithms/machinelearning/tensorflowpredicttempocnn.h

```diff
@@ -109,7 +109,7 @@ class TensorflowPredictTempoCNN : public Algorithm {
     declareParameter("input", "the name of the input nodes in the Tensorflow graph", "", "input");
     declareParameter("output", "the name of the node from which to retrieve the output tensors", "", "output");
     declareParameter("patchHopSize", "number of frames between the beginnings of adjacent patches. 0 to avoid overlap", "[0,inf)", 128);
-    declareParameter("lastPatchMode", "what to do with the last frames. Options are to `repeat` them to fill the last patch or to discard them", "{discard,repeat}", "discard");
+    declareParameter("lastPatchMode", "what to do with the last frames. `repeat` them to fill the last patch or `discard` them", "{discard,repeat}", "discard");
```

Comment:
Should we use `:` consistently? I see some changed lines go for a `.`, others for `:`.